webapp/src/ts/services/db-sync.service.ts

```diff
@@ -76,32 +77,12 @@ export class DBSyncService {
     private telemetryService:TelemetryService,
     private store:Store,
     private translateService:TranslateService,
-    private purgeService:PurgeService,
     private migrationsService:MigrationsService,
+    private replicationService:ReplicationService,
   ) {
     this.globalActions = new GlobalActions(store);
   }
 
-  private readonly DIRECTIONS = [
-    {
-      name: 'to',
-      options: {
-        filter: readOnlyFilter,
-      },
-      allowed: () => this.authService.has('can_edit'),
-      onDenied: (err?) => this.dbSyncRetryService.retryForbiddenFailure(err),
-    },
-    {
-      name: 'from',
-      options: {
-        heartbeat: 10000, // 10 seconds
-        timeout: 1000 * 60 * 10, // 10 minutes
-      },
-      allowed: () => Promise.resolve(true),
-      onChange: (replicationResult?) => this.rulesEngineService.monitorExternalChanges(replicationResult),
```

Comment:
Do we no longer need this?

Reply:
I think we do, Thanks for pointing out this todo I forgot about!